modules/gapi/test/gapi_desc_tests.cpp

```diff
@@ -236,4 +236,62 @@ TEST(GAPI_MetaDesc, Typed_Compile_MatchMetaType_Mixed)
     EXPECT_NO_THROW(cc.compile(desc1, desc2));
 }
 
+TEST(GAPI_MetaDesc, Compare_Planar)
+{
+    const auto desc0 = cv::GMatDesc{CV_8U,3,{32,32},false};
```

Comment:
Tests use square images almost everywhere. May be it makes sense to introduce rectangles {64, 32} instead to avoid potential width/height swap errors?

Reply:
Will change some tests to operate on non-square images